auth/README.md

```diff
@@ -0,0 +1,69 @@
+# Auth Package
+
+Auth package is used for request authentication and authorization for user to service
```

Comment:
The conventional Golang style is (lowercase) "package foo," right?
Unless you have a counterargument, let's look for all the "Foo Package," "Foo package," and "`Foo package`" spots in the docs and change them to "package foo."

So this will be package auth throughout.